src/dials/algorithms/shoebox/overload_checker.h

```diff
@@ -32,10 +32,10 @@ namespace dials { namespace algorithms { namespace shoebox {
       typedef Shoebox<>::float_type float_type;
 
       /**
-       * @param overload - The overload values for each pixel
+       * @param saturation - The saturation values for each pixel
```

Comment:
I appreciate that `saturation` is probably clearer than `overload` but it still seems ambiguous (without reading the code) as to whether this corresponds to the `max_trusted_value` elsewhere. At the very least a more detailed docstring/comment would be useful.